cmd/metricscollector/v1beta1/kfpv1-metricscollector/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM python:3.9-slim
```

```suggestion
FROM python:3.10-slim
```

Should we use Python v3.10 similar to other components?